spec/helpers/hyrax/collections_helper_spec.rb

```diff
@@ -80,7 +88,7 @@
       let(:test_collection_type) { FactoryBot.create(:collection_type) }
 
       it "returns the CollectionType title" do
-        expect(collection_type_label(test_collection_type.gid)).to match %r{^Title \d+$}
+        expect(collection_type_label(test_collection_type.gid)).to match %r{^Collection Type \d+$}
       end
```

Comment:
I would suggest that you check for an exact match of the title using...
```
expect(collection_type_label(test_collection_type.gid)).to eq test_collection_type.title
```

The reasoning behind this is that the title generator produces the match pattern for every collection type.  So this general match pattern could match any number of collection types and not test_collection_type specifically.  Looks like that got missed when this test was originally created.

Reply:
Ok, I can change that. I didn't look at what this was doing... I only changed it because of changing the collection type title.